modules/react/collection/lib/useListActiveDescendant.ts

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import {createElemPropsHook, slugify} from '@workday/canvas-kit-react/common';
+
+import {useCursorListModel} from './useCursorListModel';
+
+/**
+ * This elemProps hook is used for cursor navigation by using [Active
+ * Descendant](https://www.w3.org/WAI/ARIA/apg/practices/keyboard-interface/#kbd_focus_activedescendant).
+ * `aria-activedescendant` tells assistive technology which element has "focus" within the component
+ * while this single element maintains focus.
```

Comment:
ahh, is this to indicate the input to keep focus while there are focusable elements as descendant